### PR TITLE
Pal 1076 helm charts include released images

### PR DESCRIPTION
**Pull request by m78233**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -48,17 +48,17 @@
...
         <java.version>11</java.version>
+        <version.number>0.5.0</version.number>
         <maven.compiler.source>${java.version}</maven.compiler.source>
         <maven.compiler.target>${java.version}</maven.compiler.target>
         <encoding>UTF-8</encoding>
         <project.build.sourceEncoding>${encoding}</project.build.sourceEncoding>
         <project.reporting.outputEncoding>${encoding}</project.reporting.outputEncoding>
         <synthetic.data.version>0.0.4</synthetic.data.version>
-        <image.repository>gchq/palisade-</image.repository>
+        <image.repository/>
```
Comment (pd104923) [inline pom.xml:61]: This should be reverted back

File: pom.xml [hunk 2]
```diff
@@ -132,13 +132,14 @@
                 <snapshots.repository.name>Sonatype Nexus snapshot repository</snapshots.repository.name>
                 <snapshots.repository.url>https://oss.sonatype.org/content/repositories/snapshots</snapshots.repository.url>
                 <!-- Build configuration for release -->
+                <image.repository>gchq/palisade-</image.repository>
```
Comment (pd104923) [inline pom.xml:135]: This value is set above so does not need to be set here as well.

**Review by pd104923 — CHANGES_REQUESTED** [review]
A couple of changes from PAL-1077 need to be reverted back

File: pom.xml (hunk 1 above)
Comment (m78233) [reply pom.xml]: Done,

File: pom.xml (hunk 2 above)
Comment (m78233) [reply pom.xml]: Done

**Review by pd104923 — DISMISSED** [review]

File: pom.xml [hunk 3]
```diff
@@ -48,10 +48,10 @@
         <developerConnection>${scm.developer.connection}</developerConnection>
         <tag>HEAD</tag>
     </scm>
-
     <properties>
         <!-- Dependency properties -->
         <java.version>11</java.version>
+        <version.number>0.5.0</version.number>
```
Comment (dev930018) [inline pom.xml:54]: I'd also expect the project-level `<version/>` tag to be updated appropriately for this parameter:
From
```
    <version>0.5.0-${revision}</version>
```
To
```
    <version>${version.number}-${revision}</version>
```

**Review by dev930018 — CHANGES_REQUESTED** [review]

File: deployment-jvm/templates/_helpers.tpl [hunk 4]
```diff
@@ -78,3 +78,14 @@ Calculate the service config location
 {{- define "deployment-jvm.config.path" }}
 {{- printf "/usr/share/%s/config/" .Chart.Name }}
 {{- end }}
+
+{{/*
+Create the image name
+*/}}
+{{- define "deployment.image.name" }}
```
Comment (pd104923) [inline deployment-jvm/templates/_helpers.tpl:85]:
```suggestion
{{- define "deployment-jvm.image.name" }}
```

File: deployment-jvm/templates/deployment.yaml [hunk 5]
```diff
@@ -36,7 +36,7 @@ spec:
     spec:
       containers:
       - name: {{ .Chart.Name }}
-        image: "{{ .Values.global.repository }}{{ .Values.image.name }}:{{ .Values.image.base }}-{{ .Values.image.tag }}"
+        image: {{ include "deployment.image.name" .}}
```
Comment (pd104923) [inline deployment-jvm/templates/deployment.yaml:39]:
```suggestion
        image: {{ include "deployment-jvm.image.name" .}}
```

**Review by pd104923 — CHANGES_REQUESTED** [review]
Revert license changes.
Plus a couple of suggestions for deployment-jvm

File: pom.xml (hunk 3 above)
Comment (m78233) [reply pom.xml:54]: Would have preferred this approach, but this hits up agains an issue.  Both version and revision are reserved elements in Maven and can be used in the way that we have in the first example.  The element version.number does not exists at the point the version element is being assigned a value and so any attempt to us it fails.

**Comment by m78233** [comment]
Added the suggested changes and responded to the comment from dev930018

**Review by pd104923 — APPROVED** [review]

**Review by dev930018 — APPROVED** [review]

**Review by ac74475 — APPROVED** [review]

**Pull request merged**